src/test/java/xyz/vegaone/easytracking/bug/BugServiceTest.java

```diff
@@ -77,16 +81,8 @@ public void createBugTest(){
     @Test
     public void getBugTest(){
         //given
-        UserStory userStory = new UserStory();
-        userStory.setTitle(USERSTORY_TITLE);
-        userStory.setPriority(USERSTORY_PRIORITY);
-        userStory.setEstimation(USERSTORY_ESTIMATION);
-        userStory.setDescription(USERSTORY_DESCRIPTION);
-        userStory.setOwner(USERSTORY_OWNER);
-        userStory.setProjectId(PROJECT_ID);
-        userStory.setStatus(USER_STORY_STATUS);
 
-        UserStory savedUserStory = userStoryService.createUserStory(userStory);
+        UserStory savedUserStory = userStoryService.createUserStory(buildAndSaveUserStory());
 
         Bug bug = new Bug();
```

Comment:
could you also extract the this block of code into a private method? Something similar to `buildAndSaveUserStory`? Maybe `buildAndSaveBug(Long userStoryId)`, and pass the user story id as parameter.